modules/__tests__/keyframes-test.js

```diff
@@ -45,7 +49,7 @@ describe('keyframes', () => {
     expect(document.createElement).not.to.have.been.called;
     expect(document.head.appendChild).not.to.have.been.called;
 
-    var name = keyframes({});
+    var name = keyframes({}, MyComponent);
```

Comment:
can you make a copy of a simple test where you don't pass the component, just to make sure it doesn't explode?